extension/lib/main.js

```diff
@@ -46,7 +46,8 @@ exports.main = function (options, callbacks) {
     height: 107,
```

Comment:
Or we add the new help item here immediately. I wonder if this could become more flexible so we don't have to specify a height. But lets wait on that for later.